### PR TITLE
Get Feature/npm Into Develop

### PR DESCRIPTION
**Pull request by Jacob-Friesen**
There was never a proper PR on this. So I added one and will review. The only change I made, was upgrading Lodash package to fix a security vulnerability.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (Jacob-Friesen) [inline .eslintrc.js:1]: There should not be an empty line here (all eslint config file example do not have one).

File: src/services/dom-security.service.js [hunk 2]
```diff
@@ -111,56 +104,52 @@ angular.module('zerv.security')
...
+            if (elementsUnderPolicy.length) {
                 elementsUnderPolicy.forEach(function(protectedElement) {
                     applyPoliciesToProtectedElement(protectedElement.element, protectedElement.resource);
                 });
-
-            }
+            } 
+            // else {
+            // console.debug('No DOM elements are covered by the policy.');
+            // }
```
Comment (Jacob-Friesen) [inline src/services/dom-security.service.js:116]: Why was this commented code added? Either a comment should be left as to why it is needed or it should be removed.

File: src/services/dom-security.service.js [hunk 3]
```diff
@@ -111,56 +104,52 @@ angular.module('zerv.security')
...
                 protectedResource.apply(element, setting);
-                element.data('policy', protectedResource.setting);
-            }
-            else {
- //               console.debug('policy already applied to [' + protectedResource.resource.name + ']');
+                element.data('policy', setting);
             }
+            // else {
+            //     console.debug('policy already applied to [' + protectedResource.resource.name + ']');
+            // }
```
Comment (Jacob-Friesen) [inline src/services/dom-security.service.js:140]: Why was this commented code added? Either a comment should be left as to why it is needed or it should be removed.

File: src/services/dom-security.service.js (hunk 2 above)
Comment (etelmon) [reply src/services/dom-security.service.js]: For debugging. so far we do no longer need it.

File: src/services/dom-security.service.js (hunk 3 above)
Comment (etelmon) [reply src/services/dom-security.service.js]: For debugging. so far we do no longer need it.

File: .eslintrc.js (hunk 1 above)
Comment (etelmon) [reply .eslintrc.js]: removed.

**Comment by Jacob-Friesen** [comment]
I will merge this since the main develop was @etelmon and I had to just create the PR. In the future, developers of the original branches will always create the PRs and then the merge from the reviewer should not be awkward.

**Comment by etelmon** [comment]
Agreed

**Pull request merged**